users/management/commands/retire_users.py

```diff
@@ -96,6 +111,21 @@ def handle(self, *args, **kwargs):
 
             # Change user password & email
             email = user.email
+
+            if block_users:
+                hash_object = hashlib.md5(email.encode("utf-8"))
+                _, created = BlockList.objects.get_or_create(
+                    hashed_email=hash_object.hexdigest().lower()
+                )
+                if created:
+                    self.stdout.write(
+                        self.style.SUCCESS(
+                            "Email {email} is added in blocklist of MIT xPRO.".format(
```

Comment:
@asadiqbal08 Minor wording tweak:

`"Email {email} is added to the blocklist of MIT xPRO."`